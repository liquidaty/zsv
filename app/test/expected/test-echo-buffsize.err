Warning: row 1 truncated
Warning: row 1 truncated
