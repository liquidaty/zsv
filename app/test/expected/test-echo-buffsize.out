Warning: row 1 truncated
Warning: row 1 truncated
XX,XXXXXXXXXXX
XXX
XXX,XXX
